prod/westeurope/internal/api/functions_app_async/function_app/terragrunt.hcl

```diff
@@ -142,11 +130,6 @@ inputs = {
     FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
-    // Push notifications
-    AZURE_NH_HUB_NAME                       = dependency.notification_hub.outputs.name
-    NOTIFICATIONS_QUEUE_NAME                = dependency.notification_queue.outputs.name
-    NOTIFICATIONS_STORAGE_CONNECTION_STRING = dependency.notification_storage_account.outputs.primary_connection_string
```

Comment:
Have you tested that fn-async works fine without this app settings?

Reply:
ok I see there is pr reference.
Check only that fn-async starts without all these vars and references.